jena-fuseki2/apache-jena-fuseki/dist/README

```diff
@@ -6,57 +6,21 @@ Apache Jena Fuseki2 is a SPARQL server.
 Full documentation:
   http://jena.apache.org/documentation/fuseki2/
 
-This distribution contains Fuseki2 package as a WAR file ('fuseki.war'),
-to run in a webapp conatiner server such as Apache Tomcat, and also a
-self-contained jar, to run as a operation system process.
-
-The script 'fuseki' is an Linux init script to run Fuseki2 as a service.
-The file 'fuseki.service' is a systemd unit file to run Fuseki2 as a service.
-Instructions are in comments at the start of the files.
+This distribution contains Fuseki2 package as a self-contained jar, to run as a
+operation system process.
```

Comment:
operating? Confused because in Portuguese we actually use a third option: "operacional", operational :smile: